consensus/tendermint/rule_polka_nil.go

```diff
@@ -12,17 +15,7 @@ Check the upon condition on line 44:
 Line 36 and 44 for a round are mutually exclusive.
 */
 func (t *stateMachine[V, H, A]) uponPolkaNil() bool {
```

Comment:
nit: is this `t` a leftover when this was called Tendermint. What do you think of giving it an update name. It's the most nitpicks of nitpicks and to do on a separate PR. Otherwise, happy leaving the `t`

Reply:
Agree, I also wanted to do this